pokecli.py

```diff
@@ -182,10 +182,11 @@ def init_config():
     add_config(
         parser,
         load,
-        long_flag="--spin_forts",
+        long_flag="--spin",
```

Comment:
@TheSavior if in `add_config` I rewrite the long_flag like this:

```
if embedded_in:
    long_flag = "{}_{}".format(embedded_in, long_flag)
```

What you think